dummycollection.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
to be deleted

Reply:
Done

Reply:
It seems like the file is still there, maybe you have forgotten to push the commit ?